src/MyClass.java

```diff
@@ -0,0 +1,12 @@
+public class MyClass {
+    private int year = 2023;
+
+    public static int myMethod(int a, int b) {
```

Comment:
Method name should describe a sense of its work - it's better to rename the method as sumOfTwoNumbers